modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/ObjectEntryFolderResourceTest.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.delivery.resource.v1_0.test;
```

Comment:
Why is this file here? I believe it should not be. Was it generated by rest builder?

Reply:
Yes, I think it was a mistake made by the Content Management team

Reply:
I talked to them to request to send a PR to Brian